code/gradle.properties

```diff
@@ -14,7 +14,7 @@ android.enableJetifier=true
 moduleProjectName=messaging
 moduleName=messaging
 moduleAARName=messaging-phone-release.aar
-moduleVersion=2.1.1
```

Comment:
would the version bump be 2.2.0 instead as we have new api? i saw it was a minor version bump only on the ios side.

Reply:
Like I mentioned it would be a minor version update, so let's update the minor version (2.2.0), or leave as is and do it later. Let's not bump the patch version like it's done here currently.